docs/ISSUE_TEMPLATE/ISSUE_TEMPLATE_BUG_REPORT.md

```diff
@@ -1,6 +1,7 @@
 ---
-title: 'Bug report'
+title: Bug report
 labels: bug
+about: Create a report to help us improve
```

Comment:
Is this `about` necessary? Title seems self explanatory

Reply:
It's for the github subtitle =)